### PR TITLE
Fix pf? <fmt_name>

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -1319,12 +1319,14 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
 					r_core_cmd_help (core, help_detail_pf);
 				}
 			} else {
-				SdbListIter *iter;
-				Sdb *sht = core->print->formats;
-				SdbKv *kv;
-				SdbList *sdbls = sdb_foreach_list (sht, false);
-				ls_foreach (sdbls, iter, kv) {
-					r_cons_println (sdbkv_value (kv));
+				while (IS_WHITESPACE (*_input)) {
```
Comment (radare) [inline libr/core/cmd_print.c]: use r_str_trim_ro()

Comment (radare) [reply libr/core/cmd_print.c]: const char *structName = r_str_trim_ro (_input);

Comment (kazarmy) [reply libr/core/cmd_print.c]: ok 49534ca221c46735b905da48f3c2db878ab12ac4

**Comment by radare** [comment]
fuckedup branch name :D

Screenshot 2019-04-25 at 17 14 12

**Comment by kazarmy** [comment]
it looked appropriate :p

**Pull request merged**